test/new-e2e/tests/agent-metric-logs/log-agent/linux-log/file-tailing/file_tailing_test.go

```diff
@@ -218,9 +219,34 @@ func (s *LinuxFakeintakeSuite) LogRecreateRotation() {
 	t.Logf("Permissions granted for new log file.")
 
 	// Generate new logs
-	appendLog(s, "hello-world-new-content", 1)
+	s.AppendLog("hello-world-new-content", 1)
 
 	// Check intake for new logs
-	checkLogs(s, "hello", "hello-world-new-content", true)
+	s.CheckLogs("hello", "hello-world-new-content", true)
 
 }
+
+// AppendLog calls the utility function to append a log.
+func (s *LinuxFakeintakeSuite) AppendLog(content string, recurrence int) {
```

Comment:
Just curious, maybe I am not seeing the bigger picture. But what is the value of defining and using the interface methods rather than directly calling `utils.<function>` where it is needed?

Do we plan to move `LinuxFakeintakeSuite` out into it's own file, and use it in other test suits? (if so, that answers my question :) )

Reply:
In this context i don't think its needed. I was just covering for cases in the future where another suite might have to change the logic of the functions a bit so I was just thinking maybe just standardize it for all tests. Let me know if i should get rid of it and use them when needed instead 😅

Reply:
I would use explicitly the helper from the test

Reply:
Note: stacks name is based on the suite name. If you plan to use the `LinuxFakeintakeSuite` you should rename it on each test suite, or pass an explicit unique name using `WithName` to avoid conflicts across stacks in the shared cloud stack store